app/src/main/res/layout/screen_app_lock.xml

```diff
@@ -108,7 +108,15 @@
       <TextView
         android:id="@+id/applock_error"
         style="@style/Clinic.LoginPinErrorText"
-        android:text="@string/applock_error_incorrect_pin" />
+        android:text="@string/applock_error_incorrect_pin"
+        tools:visibility="visible" />
+
+      <Button
+        android:id="@+id/applock_forgotpin"
+        style="@style/Widget.AppCompat.Button.Borderless.Colored"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/applock_forgotpin" />
```

Comment:
The spacing below this button doesn't match zeplin :)

Reply:
Will fix.

Reply:
TODO: Fix in upcoming PR.

Reply:
Sure